addons/interaction/CfgVehicles.hpp

```diff
@@ -164,7 +164,7 @@ class CfgVehicles {
                 };
                 class GVAR(Gear) {
                     displayName = "$STR_ACTION_GEAR";
-                    condition = QUOTE(isNull objectParent _target && {!alive _target || {_target getVariable [ARR_2('ACE_isUnconscious',false)]} || {lifeState _target == 'INCAPACITATED'}});
+                    condition = QUOTE(!((lifeState _target) in [ARR_2('HEALTHY','INJURED')]));
```

Comment:
```suggestion
                    condition = QUOTE(!(lifeState _target in [ARR_2('HEALTHY','INJURED')]));
```
Also why remove in-vehicle check?

Reply:
I'm stupid and forgot to readd during testing.

See below for brackets.